app/uk/gov/hmrc/helptosavestub/controllers/NSIController.scala

```diff
@@ -97,7 +97,10 @@ object NSIController extends BaseController with Logging {
 
   def createAccount(): Action[AnyContent] = Action { implicit request ⇒
     val description = "create account"
-    withNSIUserInfo(description){ nsiUserInfo ⇒ handleRequest(nsiUserInfo, Created, description) }
+    withNSIUserInfo(description){ nsiUserInfo ⇒
+      val accountNumber = generateAccountNumberJson
+      handleRequest(nsiUserInfo, Created(accountNumber), description)
```

Comment:
just asking, have you tested these changes locally just to make sure the new journey works as expected ?

Reply:
Yes I have tested it locally